examples/empirical_frechet_mean_uncertainty_s3.py

```diff
@@ -0,0 +1,184 @@
+"""
+Uncertainty of the empirical Fréchet mean on the sphere
+
+The variance of the Fréchet mean FM_n of a sample of n IID
+random variables of variance Var is decreasing more slowly
+in a sphere than in a Euclidean space.
+This example computes the  modulation factor
+     alpha = Var( FM_n) / ( n * Var)
+for isotropic distributions on hyperspheres or radius 0<r<Pi
```

Comment:
For consistency reasons I'd also call `r` `theta` as it will be later in the functions

Reply:
thanks, done